packages/plugin-evm/src/actions/bridge.ts

```diff
@@ -0,0 +1,94 @@
+import { ChainId, createConfig, executeRoute, getRoutes, type Route, type Execution, ExtendedChain, Chain } from '@lifi/sdk'
+import type { WalletProvider } from '../providers/wallet'
+import type { Transaction, BridgeParams, SupportedChain } from '../types'
+import { CHAIN_CONFIGS } from '../providers/wallet'
+
+export const bridgeTemplate = `Given the recent messages and wallet information below:
```

Comment:
Can we move this template into a templates file and add all templates there for readability? Thank you!